docs/getting-started.md

```diff
@@ -95,7 +100,7 @@ If you are not inside the notes-app directory, please go in there
 
 ```
 cd notes-app
-npm install bootstrap
+npm install bootstrap@4.5.2
```

Comment:
Bootstrap's rolled forward a major version, which breaks some of the styles; forcing back to 4.5 fixes it again